fathom/src/surface/distillation.rs

```diff
@@ -407,7 +407,10 @@ impl<'interner, 'arena, 'env> Context<'interner, 'arena, 'env> {
             },
             core::Term::MetaVar(_span, var) => match self.get_hole_name(*var) {
                 Some(name) => Term::Hole((), name),
-                None => Term::Placeholder(()),
+                None => Term::Name(
+                    (),
+                    self.interner.borrow_mut().get_or_intern(format!("?{var}")),
+                ),
```

Comment:
What about updating `get_hole_name` to return `format!("{var}")` if the metavariable is unnamed?